slsSupportLib/include/sls_detector_defs.h

```diff
@@ -437,6 +439,17 @@ typedef struct {
         int64_t gateDelay3Ns{0};
         int gates{0};
     } __attribute__((packed));
+
+    /** pattern structure */
+    struct patternParameters {
+        uint64_t word[MAX_PATTERN_LENGTH];
```

Comment:
Ok if we always allocate patternParameters on the heap. Otherwise the object is quite large